css/src/tokens/themes.scss

```diff
@@ -147,15 +149,17 @@ $themes: (
 		'score-medium': $palette-yellow-40,
 		'score-high-off': $palette-green-opacity-70,
 		'score-high': $palette-green-40,
-		'hyperlink': $palette-blue-30,
-		'primary-base': $palette-blue-30,
-		'primary-background': $palette-blue-80,
-		'primary-background-glow-high-contrast': $palette-blue-80,
-		'primary-dark': $palette-blue-20,
-		'primary-hover': $palette-blue-40,
-		'primary-active': $palette-blue-50,
+		'hyperlink': $palette-blue-90,
```

Comment:
Suggestion only: perhaps revert all dark theme changes and swing back to that as a separate PR? There's enough to think through going one theme and one semantic color at a time. The mix of old and new palette may make things harder to think about as well.

Reply:
Separate as in a PR with light theme changes, and a second PR with only dark theme changes? or a second PR with both light and dark theme changes? In parallel or in succession? I'm open to having separate PRs if that makes it easier to review.

The situation I'd like to avoid is being caught in a state where we merge the light theme, knowing that some colors don't work temporarily in dark/high-contrast theme. We address those themes in a separate PR, but the time gap between the two PRs may be long and we end up needing to ship something else (timeline component for example) during that in-between state. This situation is avoidable if we make sure to merge all 2 or 3 PRs at the same time and deal with the potential merge conflicts.

Reply:
The change to dark theme is mainly to point to the new names from the old palette. There were two actual value changes that I've reverted. The remaining changes are the two new items, which need to have a value for every theme.